go/tools/builders/env.go

```diff
@@ -133,18 +133,14 @@ func (e *env) runCommand(args []string) error {
 	cmd := exec.Command(args[0], args[1:]...)
 	// Redirecting stdout to stderr. This mirrors behavior in the go command:
 	// https://go.googlesource.com/go/+/refs/tags/go1.15.2/src/cmd/go/internal/work/exec.go#1958
-	cmd.Stdout = os.Stderr
-	cmd.Stderr = os.Stderr
-	return runAndLogCommand(cmd, e.verbose)
+	return runAndLogCommandOutput(cmd, os.Stderr, os.Stderr, e.verbose)
 }
 
 // runCommandToFile executes a subprocess and writes the output to the given
 // writer.
 func (e *env) runCommandToFile(w io.Writer, args []string) error {
 	cmd := exec.Command(args[0], args[1:]...)
-	cmd.Stdout = w
-	cmd.Stderr = os.Stderr
-	return runAndLogCommand(cmd, e.verbose)
+	return runAndLogCommandOutput(cmd, w, os.Stderr, e.verbose)
```

Comment:
Let's be careful that `runCommandToFile` does not relativize the output. It should just call `runAndLogCommand`.

Reply:
Apologies, missed that requirement. Updated